tests/gamification/test_coins_made_real.py

```diff
@@ -0,0 +1,278 @@
+            users_count=0,
+            users_processed=[],
+            users_skipped=[],
+            task_data={'data': 'data'}).save()
+
+        fake_type = FakeType({})
+        task = fake_type.task_model(
+            id='task1',
+            task_type=fake_type.type_name,
+            batch=self.GAME_BATCH,
+            closed=False,
+            users_count=2,
+            users_processed=[],
+            users_skipped=[],
+            task_data={'data': 'data'}).save()
+        TASKS_TYPES[fake_type.type_name] = fake_type
+        TASKS_TYPES[another_type.type_name] = another_type
+
+        # when user one finishes the task not to be closed
+        another_type.work_session_manager.start_work_session(
+            another_task, self.USER_ONE.id)
+        another_type.on_task_done(
+            self.USER_ONE, another_task.id, {'result': 'result'})
+        # and user two finishes the task to be closed and to close the batch
+        fake_type.work_session_manager.start_work_session(
+            task, self.USER_TWO.id)
+        fake_type.on_task_done(
+            self.USER_TWO, task.id, {'result': 'result'})
+        # then only the user two shall get actual coins
+        usm_one = UserStateModel.get_or_create_by_user(self.USER_ONE)
+        usm_two = UserStateModel.get_or_create_by_user(self.USER_TWO)
+
+        self.assertEqual(usm_one.potential_coins, Decimal(20))
+        self.assertEqual(usm_one.actual_coins, Decimal(0))
+        self.assertEqual(usm_two.potential_coins, Decimal(0))
+        self.assertEqual(usm_two.actual_coins, Decimal(1))
+
+    def test_materialize_all_participated(self):
+        fake_type = FakeType({})
+        task = fake_type.task_model(
+            id='task1',
+            task_type=fake_type.type_name,
+            batch=self.GAME_BATCH,
+            closed=False,
+            users_count=1,
+            users_processed=[],
+            users_skipped=[],
+            task_data={'data': 'data'}).save()
+        TASKS_TYPES[fake_type.type_name] = fake_type
+
+        # when user one finishes the task
+        fake_type.work_session_manager.start_work_session(
+            task, self.USER_ONE.id)
+        fake_type.on_task_done(
+            self.USER_ONE, task.id, {'result': 'result'})
+        # and user two finishes the task
+        fake_type.work_session_manager.start_work_session(
+            task, self.USER_TWO.id)
+        fake_type.on_task_done(
+            self.USER_TWO, task.id, {'result': 'result'})
+        # then both users shall get actual coins
+        usm_one = UserStateModel.get_or_create_by_user(self.USER_ONE)
+        usm_two = UserStateModel.get_or_create_by_user(self.USER_TWO)
+
+        self.assertEqual(usm_one.potential_coins, Decimal(0))
+        self.assertEqual(usm_one.actual_coins, Decimal(1))
+        self.assertEqual(usm_two.potential_coins, Decimal(0))
+        self.assertEqual(usm_two.actual_coins, Decimal(1))
+
+
+if __name__ == '__main__':
```

Comment:
Cannot you just use setup.py test -m/-s option to run tests in particular module?